sdk/react/bottomOverlay.js

```diff
@@ -117,13 +117,20 @@ var BottomOverlay = React.createClass({
     var topOffset = this._renderTopOffset(scrubberSize);
     var leftOffset = this._renderLeftOffset(scrubberSize, percent, progressBarWidth);
     var positionStyle = {top:topOffset, left:leftOffset};
+    var scrubberStyle = this._customizeScrubber();
 
     return (
-      <View style={[styles.progressScrubber, positionStyle,{width:scrubberSize, height:scrubberSize}]}>
```

Comment:
yeah the style in the .json file.  if we can delete it, we should.  alternatively, you can keep it (only remove the color parts), and define the coloring styles inline.  either way is good